testing/alacritty/APKBUILD

```diff
@@ -17,25 +17,22 @@ source="$pkgname-$pkgver.tar.gz::https://github.com/jwilm/$pkgname/archive/v$pkg
 	ignore-ft-fc-tests-font_mated.patch
 	ignore-x11-tests-clipboard_works.patch
 	"
-builddir="$srcdir/$pkgname-$pkgver"
```

Comment:
What's this?

Reply:
$srcdir/$pkgname-$pkgver is the default value for builddir